setup.py

```diff
@@ -8,21 +8,20 @@
 DOCLINES = __doc__.split("\n")
 
 setup(
-    name='crank',
+    name='torsiondrive',
     author='Yudong Qiu, Lee-Ping Wang',
     description=DOCLINES[0],
     long_description="\n".join(DOCLINES[2:]),
     version=versioneer.get_version(),
     cmdclass=versioneer.get_cmdclass(),
-    packages=['crank', "crank.tests"],
+    packages=['torsiondrive', "torsiondrive.tests"],
     entry_points={'console_scripts': [
-        'crank-launch = crank.launch:main',
-        'crank-api = crank.crankAPI:main',
+        'td-launch = torsiondrive.launch:main',
```

Comment:
Any reason to shorted this here? `torsiondrive-launch` is very clear and not too long I think.

Reply:
Updated 👍